pkgs/development/misc/resholve/resholve.nix

```diff
@@ -50,5 +51,6 @@ python27.pkgs.buildPythonApplication {
     license = with licenses; [ mit ];
     maintainers = with maintainers; [ abathur ];
     platforms = platforms.all;
+    knownVulnerabilities = [ "resholve depends on python27 that is EOL" ];
```

Comment:
```suggestion
    knownVulnerabilities =  [ ''
      resholve depends on python27 (EOL). While it's safe to
         run on trusted input in the build sandbox, you should
         avoid running it on untrusted input.
    '' ];
```

Would like this to be specific enough for people who aren't up on the python2 situation evaluate the risk well enough to decide whether or not to permit it. WDYT?